### PR TITLE
Defined Update URL properties

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
[Orchestrator to update cases with Supplementary Evidence](https://tools.hmcts.net/jira/browse/BPS-833)

### Change description ###
Define properties for Update URL in client services.
It is separate from transformation URL (though at the moment the values are the same) to make it possible for the services to implement transform and update functionality in a flexible way, potentially on different servers.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -79,6 +79,10 @@ variable "transformation_url_probate" {
   default = ""
 }
 
+variable "update_url_bulkscan" {
+  default = "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```
Comment (doncem) [inline infrastructure/variables.tf]: there's no need for this variable - can just simply assign directly in main app settings

Comment (rkondratowicz) [reply infrastructure/variables.tf]: it's consistent with transformation though

Comment (doncem) [reply infrastructure/variables.tf]: yes. it is. and it has no value in it. we moved to aks, aks defined in all envs. this terraform change is legacy only applied in old demo environment. just keep in consistent and define in main.tf. if endpoint doesn't work it'll just be smth azure cloud provides ;)

File: charts/bulk-scan-orchestrator/values.preview.template.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ java:
     CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
     TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
+    UPDATE_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```
Comment (lgonczar) [inline charts/bulk-scan-orchestrator/values.preview.template.yaml]: Consistent with `TRANSFORMATION_URL_...`, so this PR is good, but I think we will need another PR to change the naming of these environment variables to sth like `CASE_TRANSFORMATION_URL_{service}` and `CASE_UPDATE_URL_{service}` to make it clearer what they're about.

**Pull request merged**